src/main/java/com/unigrad/funiverseauthenservice/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package com.unigrad.funiverseauthenservice.security;
+
+import com.unigrad.funiverseauthenservice.security.jwt.JwtAuthenticationFilter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Configuration
+@EnableWebSecurity
+@RequiredArgsConstructor
+public class SecurityConfiguration {
+
+  private final JwtAuthenticationFilter jwtAuthFilter;
```

Comment:
Create a bean
```
  @Bean
    public JwtAuthenticationFilter jwtAuthenticationFilter() {
        return new JwtAuthenticationFilter();
    }
```
also for `AuthenticationProvider`

Reply:
Currently, if I add it, I will get an error due to the annotation of lombok, I will check and fix this comment later

Reply:
Currently, if I add it, I will get an error due to the annotation of lombok, I will check and fix this comment later